addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -476,20 +476,41 @@ var ListController = BasicController.extend({
      * @returns {Promise}
      */
     _saveMultipleRecords: function (recordId, node, changes) {
-        var fieldName = Object.keys(changes)[0];
-        var value = Object.values(changes)[0];
         var recordIds = _.union([recordId], this.selectedRecords);
+        const nodes = [node];
+        const isDateRangeWidget = node.attrs.widget === 'daterange';
+        let relatedFieldName;
+        if (isDateRangeWidget) {
+            const options = this.model.get(recordId).fieldsInfo.list[node.attrs.name].options;
+            let fieldName;
+            if (options.hasOwnProperty('related_end_date')) {
+                fieldName = options.related_end_date;
+            } else if (options.hasOwnProperty('related_start_date')) {
+                fieldName = options.related_start_date;
+            }
```

Comment:
Same as above about the fact that we want to bound the logic to the daterange or wether we want to have something more generic. I really don't know, I am just asking the question 🙂 I guess the JS framework will have their opinion on this.